### DISCUSSION TITLE
Direct URLs allow me to download files without authentication

### DISCUSSION THREAD

**Discussion by GeistFighter** [authelia/authelia#7829] [Q&A] [answered]
Hello everyone,

I have a small question, I have set up Authelia under my NGINX via the APT package, I have done the configuration and I have set up the documentation configuration:
[https://www.authelia.com/integration/proxies/nginx/#proxyconf](https://www.authelia.com/integration/proxies/nginx/#proxyconf)
[https://www.authelia.com/integration/proxies/nginx/#authelia-locationconf](https://www.authelia.com/integration/proxies/nginx/#authelia-locationconf)
[https://www.authelia.com/integration/proxies/nginx/#authelia-authrequestconf](https://www.authelia.com/integration/proxies/nginx/#authelia-authrequestconf)

However, when I go to my site https://home.exemple.com/assets/boxicons/fonts/boxicons.woff2 the file is downloaded, and no authentication is required, whereas if I go to https://home.exemple.com/assets/boxicons/fonts/ it redirects me to https://auth.exemple.com/?rd=https%3A%2F%2Fhome.exemple.com%2Fassets%2Fboxicons%2Ffonts%2F&rm=GET.

Why isn't the direct link blocked on connection?

**Comment by james-d-elliott** [comment, marked as answer]
It will be blocked if the request is made via the proxy. Can you show the debug logs and access control config? Have you tried via an incognito browser to avoid caches? Does the browser show the content as cached or not cached?

**Reply by GeistFighter** [reply]
Hello @james-d-elliott,

Thanks for your reply!

Indeed, it was certainly a cache problem on my second browser, in private browsing and after deleting the browser cache, it does ask for authentication to access the file.

Thanks for the information, do you still want logs for investigation?

**Reply by james-d-elliott** [reply]
Nope, that's fine. I suspected you'd find the cause or show the logs and I'd be able to give you more pointed advice.
